django/db/backends/postgresql/operations.py

```diff
@@ -105,10 +108,7 @@ def datetime_trunc_sql(self, lookup_type, sql, params, tzname):
     def time_extract_sql(self, lookup_type, sql, params):
         if lookup_type == "second":
             # Truncate fractional seconds.
-            return (
-                f"EXTRACT(%s FROM DATE_TRUNC(%s, {sql}))",
-                ("second", "second", *params),
-            )
+            return (f"EXTRACT(SECOND FROM DATE_TRUNC('second', {sql}))", params)
```

Comment:
Can't the `DATE_TRUNC`'s `%s` be kept?

```suggestion
            return f"EXTRACT(SECOND FROM DATE_TRUNC(%s {sql}))", ("second", *params)
```